frontend/src/components/VariantListPage/VariantsTable.tsx

```diff
@@ -634,6 +653,85 @@ const VariantsTable: FC<VariantsTableProps> = ({
     sortedVariants.reverse();
   }
 
+  const ROW_HEIGHT = 70;
+
+  const VariantRow = ({
+    index: dataRowIndex,
+    data: { columns, data },
+    style,
+  }: {
+    index: number;
+    data: {
+      columns: ColumnDef[];
+      data: any;
+    };
+    style: any;
+  }) => {
+    const rowData = data[dataRowIndex];
+    const rowIndex = dataRowIndex + 1;
+    return (
+      <Tr
+        key={rowIndex}
+        sx={{
+          display: "flex",
+          flexDirection: "row",
+          alignItems: "stretch",
+          boxSizing: "border-box",
+          height: `${ROW_HEIGHT}px`,
+          background: dataRowIndex % 2 === 1 ? "initial" : "#edf2f7",
+        }}
+        style={style}
+      >
+        {includeCheckboxColumn && (
```

Comment:
More content then code -- it appears as though the moving of this component to use `FixedSizeList` has caused some funny things size, alignment and coloring wise:

![Screenshot 2024-09-09 at 16 02 27](https://github.com/user-attachments/assets/48794d53-714f-4660-a432-a3bd26d065cb)

- The rows appear to have grown in height, it may be useful to try and keep the rows smaller if possible to utilize the space in a similar way as before
- Wrapping the rows in the row from the fixed size list appears to have caused the contents of the cells to align to the top, rather than being centered
- The "+" button used to add a note to a variant now uses the same color background as the darker rows, two possible solutions are to not have alternating colored rows, or to also have alternating colored buttons. Personally I slightly prefer to just have all the same color row to keep things as similar as to before as possible, but I'll defer to you here

I realize some of these bullet points are likely here due to the implementation of the dashboard list, particularly with the justification and sizing. That is a slightly different use case where space is at a bit less of a premium (the table takes up more of the screen), hence the request for some differences in aesthetics!